docs/generate_examples.py

```diff
@@ -214,6 +219,27 @@ def main():
         sig = f"{aug_name}({', '.join([str(a) for a in args])}, p=1.0)"
         print(f"Generated image example for {aug_name}. {sig}")
 
+    mask_augmentations_list = {"RandomTransplantation": (([0],), 0)}
+    # ITERATE OVER THE TRANSFORMS
+    for aug_name, (args, seed) in mask_augmentations_list.items():
+        img_in = torch.cat([img1, img2])
+        mask_in = torch.cat([torch.zeros_like(mask2), mask2])
+
+        # dynamically create the class instance
+        cls = getattr(mod, aug_name)
+        aug = cls(*args, p=1.0)
+        # set seed
+        torch.manual_seed(seed)
+        # apply the augmentation to the image and concat
+        img_aug, _ = aug(img_in, mask_in)
+
+        output = torch.cat([img_in[0], img_in[1], img_aug[0]], dim=-1)
+        # save the output image
+        out_np = K.utils.tensor_to_image((output * 255.0).byte())
+        cv2.imwrite(str(OUTPUT_PATH / f"{aug_name}.png"), out_np)
```

Comment:
you can use `kornia.io.write_image` here: https://github.com/kornia/kornia/blob/master/kornia/io/io.py#L119

Reply:
@edgarriba you know if we have some issues tracking the rewrite of this `generate examples` (aside the #2443)?

this script is worth a total rewrite, using some new functions such as `kornia.io.write_image` and simplifying as said in #2443

Reply:
Should I leave this as-is for now since you intend to rewrite the whole script anyway? Otherwise I can of course switch to `kornia.io.write_image`.

Reply:
> Should I leave this as-is for now since you intend to rewrite the whole script anyway? 

yes, we can open an issue later to track this :)